src/Http/Controllers/Admin/XlsformTemplateCrudController.php

```diff
@@ -166,4 +193,15 @@ public function setupShowOperation(): void
 
         $this->setupListOperation();
     }
+
+
+    // Select button, divert to a fully customized blade view file
+    public function select()
+    {
+        logger("XlsformTemplateCrudController.select() starts...");
+
+        $entry = CRUD::getCurrentEntry();
+
+        return view('xlsformtemplate.select-odk-variables', ['entry' => $entry]);
```

Comment:
Similar to the comment above about namespacing of views, this needs to include the `odk-link::` namespace.

Reply:
Dave, Thanks for your advice, it is now working.